### ISSUE TITLE
Brace style conflicting with single line per selector rule

### ISSUE THREAD

**Issue by abpetkov** [sasstools/sass-lint#917]
So in my `.sass-lint.yml` I have these two rules:

```
brace-style: 2
single-line-per-selector: 2
```
Using version 1.9.1 and running the linter via CLI.

Now, the purpose of this obviously is to allow single line rules, since our styling standards allow selectors with a single property to be written in a single line, but if more than one selector share a single property, we need them to be on a single line each, e.g.:

```
// OK
.selector { padding: 0; }

// OK
.selector,
.another-selector,
.yet-another-selector { padding: 0; }

// NOT OK
.selector, .another-selector, .yet-another-selector { padding: 0; }
```

The second `OK` example, throws `error    Closing brace must be on a new line                             brace-style`, while I expect it not to, since we need every selector to appear on a new line, but the actual body to be inline.

So it looks to me like it was intended to work this way, my question is: is it possible to make the necessary improvements, so that the `brace-style` rule respects the `single-line-per-selector` rule and doesn't throw an error? Or just have the option to allow this style?

**Comment by brianO3world** [comment]
I'm experiencing this same issue, specifically for:

```
// OK
.selector,
.another-selector,
.yet-another-selector { padding: 0; }
```

Any word if these properties can work together to allow the above the be valid?

Thanks
Brian
